tools/schema/location.json

```diff
@@ -0,0 +1,38 @@
+{
+    "title": "location",
```

Comment:
I don't think this should be called "location" because "location" already means something else (position + world). Maybe "Area"?

Reply:
This is taken straight out of the prerequisites file; I'm not sure I want to change it and risk needing to update the prerequisite code now, especially if we're considering reworking this to handle all bounded regions in our code later.

Reply:
Crap, I didn't realize I was the one who screwed this up to begin with. Ok, agreed - let's not rename this. Damn.